src/transformers/models/mobilevit/feature_extraction_mobilevit.py

```diff
@@ -151,3 +154,46 @@ def __call__(
         encoded_inputs = BatchFeature(data=data, tensor_type=return_tensors)
 
         return encoded_inputs
+
+    def post_process_semantic_segmentation(self, outputs, target_sizes: Union[TensorType, List[Tuple]] = None):
```

Comment:
Could this be added to the Mixin instead of the FeatureExtractor?

Reply:
@hollance  I agree with this but I'd prefer to do this after making sure (1) all post-processing methods of all segmentation models have consistent input arguments and naming and (2) `ImageSegmentationPipeline` supports all available segmentation models rather than just DETR and MaskFormer.